src/lib/Form/DataTransformer/LocationsTransformer.php

```diff
@@ -27,19 +29,41 @@ public function __construct(LocationService $locationService)
         $this->locationService = $locationService;
     }
 
+    /**
+     * Transforms a domain specific Location objects into a Location's ID comma separated string.
+     *
+     * @param mixed $value
+     * @return array|mixed|string
```

Comment:
As far as I understand it always return an `array` so the `@return` tag is wrong here.